__tests__/consRouter.test.js

```diff
@@ -0,0 +1,199 @@
+const supertest = require('supertest');
+
+const server = require('../api/server.js');
+
+const db = require('../database/dbConfig');
+
+const {
+  find,
+  findById,
+  insert,
+  update,
+  remove
+} = require('../Models/consModel');
+
+describe('Cons Routes', () => {
+  beforeEach(async () => {
+    await db("conservationists").truncate();
+  });
+
+  describe('GET /api/cons', () => {
+    it('should respond with 200', async () => {
+      const res = await supertest(server).get('/api/cons/');
+      expect(res.status).toBe(200);
+    });
+  });
+  it('should return an array', async () => {
+    let result;
+    await supertest(server)
+      .get('/api/cons')
+      .then(res => result = res.body);
+    expect (result.cons.length).toEqual(0)
+  });
+  describe("GET /api/cons/:id", () => {
+    it("responds 200", async () => {
+      let save = {
+        user_id: 1,
+        lat: 1234,
+        lon: 5678,
+        address: "123 4th St."
+      };
+
+      await db("saves").insert(save);
+
+      let { id } = await db("saves")
+        .where("lat", 1234)
+        .first();
+
+      await supertest(server)
+        .get(`/saves/${id}`)
+        .set("Authorization", token)
```

Comment:
I'm assuming the token will be set later? Or am I missing a variable somewhere?

Reply:
Lol, forgot about those. I pasted those tests in from an old build week project in case we wanted to repurpose some of them. They are... not accurate to our current project :D

Reply:
I was wondering what the 'saves' table was 😛. Ok cool as long as we know the Token hasnt been set yet, thats all that matters!